drivers/hmis/app/graphql/types/hmis_schema/service.rb

```diff
@@ -0,0 +1,43 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::Service < Types::BaseObject
+    description 'HUD Service'
+    field :id, ID, null: false
+    field :enrollment, Types::HmisSchema::Enrollment, null: false
+    field :client, HmisSchema::Client, null: false
+    field :date_provided, GraphQL::Types::ISO8601Date, null: false
+    field :record_type, HmisSchema::Enums::RecordType, null: false
+    field :type_provided, HmisSchema::Enums::ServiceTypeProvided, null: false
+    field :other_type_provided, String, null: true
+    field :moving_on_other_type, String, null: true
+    field :sub_type_provided, HmisSchema::Enums::ServiceSubTypeProvided, null: true
```

Comment:
This isn't working for me, as mentioned in Slack, here's an example:

Record in the database:
```
#<Hmis::Hud::Service:0x0000ffff6fdd3e10
 ServicesID: "2360155",
 EnrollmentID: "1376952",
 PersonalID: "28815d7a5372c2125e339e91d644350f",
 DateProvided: Thu, 10 Dec 2020,
 RecordType: 144,
 TypeProvided: 3,
 OtherTypeProvided: nil,
 SubTypeProvided: 2,
 FAAmount: nil,
 ReferralOutcome: nil,
 DateCreated: Thu, 24 Dec 2020 15:19:08.000000000 EST -05:00,
 DateUpdated: Thu, 24 Dec 2020 15:19:08.000000000 EST -05:00,
 UserID: "nelson",
 DateDeleted: nil,
 ExportID: "879da0a2df463f2e84e0852434052b5f",
 data_source_id: 4,
 id: 202378,
 source_hash:
  "f562072a5b23967f720c4332c39dafc9ef5a9690c395e2b0d2de1cd0c461a353",
 pending_date_deleted: nil,
 MovingOnOtherType: nil>
```

The TypeProvided is valid (in V2.2 SSVFServices), and the SubTypeProvided is valid (in V2.A SSVFSubType3).

I get this error when querying the Enrollment that the above service belongs to:
```
{
  "errors": [
    {
      "message": "`Service.subTypeProvided` returned `2` at `enrollment.services.nodes.0.subTypeProvided`, but this isn't a valid value for `ServiceSubTypeProvided`. Update the field or resolver to return one of `ServiceSubTypeProvided`'s values instead.",
      "backtrace": [
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/schema/enum.rb:143:in `coerce_result'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:684:in `continue_field'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:516:in `block (3 levels) in evaluate_selection_with_args'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:925:in `after_lazy'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:513:in `block (2 levels) in evaluate_selection_with_args'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:826:in `call_method_on_directives'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:498:in `block in evaluate_selection_with_args'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:925:in `after_lazy'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:436:in `evaluate_selection_with_args'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:425:in `evaluate_selection'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/execution/interpreter/runtime.rb:358:in `block (2 levels) in evaluate_selections'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/dataloader.rb:181:in `block in run'",
        "/bundle/ruby/2.7.0/gems/graphql-2.0.9/lib/graphql/dataloader.rb:303:in `block in spawn_fiber'"
      ]
    }
  ],
  "data": {}
}
```

Reply:
@gigxz Ah, I must have missed a resolver. The value for that should be `144:3:2`, but it looks like it's just sending `2`. I'll fix it

Reply:
@gigxz Fixed now